WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -116,68 +125,48 @@ public WpkhDescriptors GetOutputDescriptors(string password, Network network)
 		return WpkhOutputDescriptorHelper.GetOutputDescriptors(network, MasterFingerprint.Value, GetMasterExtKey(password), AccountKeyPath);
 	}
 
-	[JsonProperty(Order = 1)]
-	[JsonConverter(typeof(BitcoinEncryptedSecretNoECJsonConverter))]
 	public BitcoinEncryptedSecretNoEC EncryptedSecret { get; }
```

Comment:
What will be the order then? The property order in the class?

Reply:
The order doesn't matter if we don't have to keep compatibility. Anyway, this is not the final code, in fact i cannot even start.